app/src/main/java/com/ataulm/artcollector/Navigation.kt

```diff
@@ -23,14 +26,15 @@ fun artistGalleryIntent(artistId: String): Intent {
     return intent(ARTIST_GALLERY, uri)
 }
 
-fun paintingIntent(artistId: String, paintingId: String): Intent {
+fun paintingIntent(artistId: String, paintingId: String, imageUrl: String? = null): Intent {
```

Comment:
**optional** intent extras mean that we can support nice animations when navigating internally, but can still support "cold" entries with app links.